src/Microsoft.PowerShell.Commands.Utility/commands/utility/Get-Error.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-windows`
SSHConnection parameter hashtable type conversions.SSHConnection can convert PSObject to string
```powershell
Expected an exception, with FullyQualifiedErrorId '2100,PSSessionOpenFailed' to be thrown, but the FullyQualifiedErrorId was 'Argument,Microsoft.PowerShell.Commands.NewPSSessionCommand'. from D:\a\1\s\test\powershell\engine\Remoting\SSHRemotingCmdlets.Tests.ps1:51 char:30
    + ? iptBlock = {New-PSSession -ErrorAction Stop -SSHConnection @{ Port =  ?
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Remoting\SSHRemotingCmdlets.Tests.ps1: line 58
58:         { & $scriptBlock } | Should -Throw -ErrorId '2100,PSSessionOpenFailed'
```
SSHConnection parameter hashtable type conversions.SSHConnection can convert PSObject to int
```powershell
Expected an exception, with FullyQualifiedErrorId '2100,PSSessionOpenFailed' to be thrown, but the FullyQualifiedErrorId was 'Argument,Microsoft.PowerShell.Commands.NewPSSessionCommand'. from D:\a\1\s\test\powershell\engine\Remoting\SSHRemotingCmdlets.Tests.ps1:52 char:30
    + ? iptBlock = {New-PSSession -ErrorAction Stop -SSHConnection @{ Port =  ?
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Remoting\SSHRemotingCmdlets.Tests.ps1: line 58
58:         { & $scriptBlock } | Should -Throw -ErrorId '2100,PSSessionOpenFailed'
```